README.md

```diff
@@ -27,6 +28,78 @@ Rich IntelliSense (completions) for `Dockerfile` and `docker-compose.yml` files:
 
 ![IntelliSense for DockerFiles](images/intelliSense.gif)
 
+## Debugging .NET Core ASP.NET (Preview)
+
+To debug a .NET Core ASP.NET application running in a Linux Docker container, add a Docker .NET Core launch configuration:
```

Comment:
Be more explicit about saying Windows is not currently supported (preferably verify from code).

Reply:
Added not to `README.MD` and prerequisite check for a Linux Docker host.